go/ephemeral/common.go

```diff
@@ -4,13 +4,21 @@ import (
 	"context"
 	"fmt"
 	"sort"
-	"time"
 
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
 )
 
-const KeyLifetimeSecs = keybase1.Time(time.Hour * 24 * 7) // one week
+// Keys last at most one week
+const KeyLifetimeSecs = keybase1.Time(60 * 60 * 24 * 7) // one week
```

Comment:
I think these units are still off. The `keybase1.Time` type is an `int64` of milliseconds, but the number we're casting is in seconds. I think in general we should avoid doing a cast, and try to use the conversion functions. If that makes it impossible to have a `const`, then we could consider exposing helper constants from inside of `keybase1`, similar to how the `time` module does?

Reply:
off everywhere haha. fixing